### ISSUE TITLE
Grouping multiday events on top of list view

### ISSUE THREAD

**Issue by KarlBaumann** [fullcalendar/fullcalendar#4293]
Hi!

Is there a way to show all multiday events on the top of list view?

I have some multiday events that last half a year and it does not look good if they are being repeated in each day.

What would be the best approach to make this work?

**Comment by acerix** [comment]
It's not built-in, but you should be able to do that with a custom view.

This could be useful a useful feature though.  Would "multi-day" be any event that occurs on more than one date in the time period that is being displayed?

**Comment by KarlBaumann** [comment]
yes. The event date would intersect with a period displayed in a view.

**Comment by docpitz** [comment]
Have you found a solution? I have searched the whole Internet and unfortunately found nothing suitable.

**Comment by yoancutillas** [comment]
Hello, I am interested in this feature too. It would be great as it can be misleading to see multiple events when there is actually only one.

Before:
![fullcalendar-list-multiday-events](https://user-images.githubusercontent.com/9415030/81163544-3a558580-8f8f-11ea-8a49-b7811aeb28ac.PNG)

After (mockup):
![fullcalendar-list-multiday-events-proposal](https://user-images.githubusercontent.com/9415030/81163553-3de90c80-8f8f-11ea-8028-550606e222c2.PNG)

**Comment by bedscent** [comment]
I'm interested in this as well, if anyone has any idea's that would be great to hear about them.

I would like them to be listed similar to above, but all events that start in say April are listed under the month of April heading in the list year option

**Comment by rdupont55** [comment]
I would love something like this.  Showing a 3-day event 3 times in a row does not accurately communicate the event.  I can see this being an issue on a calendar with multiple events across several days, but an for an application in which multiple non-overlaping multi-day events exist it would be nice to simply list Start - End: Event Name
